locales/it-IT/messages.po

```diff
@@ -422,8 +452,8 @@ msgstr "Se richiesto, deve apparire almeno una volta nella selezione"
 msgid "If set, it causes the order of the child elements to be randomized, otherwise it uses the order in which the child elements are defined."
 msgstr "Se impostato, provoca l'ordine degli elementi figlio da randomizzare, altrimenti utilizza l'ordine in cui sono definiti gli elementi figlio."
 
-msgid "If the candidate can skip the item, without submitting a response."
-msgstr "Se il candidato può ignorare l'item, senza presentare una risposta."
+msgid "If the candidate can skip the item, without submitting a response (default is true)."
+msgstr ""
```

Comment:
Italian translation disappeared.

Reply:
nope, it's not the same `(default is true).` Need to be retranslated in all languages